changelog.md

```diff
@@ -10,6 +10,11 @@ The idea is to track end-user facing changes as they occur.*
   - Now using Microsoft.Extensions.DepedencyInjection.ServiceProvider as the default service provider if the user does not override it.
   - Grains are still not being injected automatically unless the user opts in by specifying his own Startup configuration that returns a service provider.
 - Updated Azure Storage dependency to 7.0.0 #1968
+- The SQL Server specific persistence provider rewritten to support any ADO.NET backend. Configuration is now done as for any (programmatic) persistence provider
```

Comment:
I suggest something like:
A brand new ADO.NET based storage provider which replaces the current SQL Server specific storage provider. The change is not backwards compatible and removes the use of [Elastic Sharding](https://github.com/Azure/elastic-db-tools) tooling (likely be replaced later with Orleans sharding provider). More information in #1682 and in [Grain Persistence](http://dotnet.github.io/orleans/Getting-Started-With-Orleans/Grain-Persistence) documentation.